docs/versionhistory.rst

```diff
@@ -12,26 +12,30 @@ This library adheres to `Semantic Versioning 2.0 <http://semver.org/>`_.
 
   * ``current_time()``
   * ``current_effective_deadline()``
+  * ``get_current_task()``
```

Comment:
For even more improved fake-await-ery we could add ```
def __await__(self):
    if False: yield
    warn("Don't do this")
    return self
```
method to `ansio.TaskInfo`. That'd fix `get_current_task`. A list with that method tacked on is almost as simple, for `get_running_tasks`.

Reply:
Ok, that's fair.